vasyl_varyshchuk/Week_1/task_1/test_get_max_value.py

```diff
@@ -0,0 +1,8 @@
+from max_of_list import get_max_value
+
+
+def test_get_max_value():
+
+    assert get_max_value(['5', 'some text', 'nine', '9', '3.5', '12k']) == 9
+
+    assert get_max_value(['hkk', 'one', 'fgdg']) == None
```

Comment:
I expected to have these tests as a separate test but in the same module.

Reply:
Also, please use some sensitive values for each element.
Try to avoid values like `hkk` and `fgdg`.